bundle/DependencyInjection/Configuration.php

```diff
@@ -0,0 +1,42 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Netgen\Bundle\IbexaAdminUIExtraBundle\DependencyInjection;
+
+use Symfony\Component\Config\Definition\Builder\ArrayNodeDefinition;
+use Symfony\Component\Config\Definition\Builder\TreeBuilder;
+use Symfony\Component\Config\Definition\ConfigurationInterface;
+
+final class Configuration implements ConfigurationInterface
+{
+    private string $rootNodeName;
+
+    public function __construct(string $rootNodeName)
+    {
+        $this->rootNodeName = $rootNodeName;
+    }
+
+    public function getConfigTreeBuilder(): TreeBuilder
+    {
+        $treeBuilder = new TreeBuilder($this->rootNodeName);
+        $rootNode = $treeBuilder->getRootNode();
+
+        $this->addShowExternalSiteaccessUrls($rootNode);
+
+        return $treeBuilder;
+    }
+
+    private function addShowExternalSiteaccessUrls(ArrayNodeDefinition $nodeDefinition): void
+    {
+        $nodeDefinition
+            ->treatFalseLike(['enabled' => false])
+            ->treatTrueLike(['enabled' => true])
+            ->treatNullLike(['enabled' => false])
+                ->children()
+                    ->booleanNode('show_external_siteaccess_urls')
```

Comment:
Document the setting with `info()`.

Reply:
fixed in https://github.com/netgen/ibexa-admin-ui-extra/pull/2/commits/9f950749374566c7e43cde1df1b72c3351ce5e50